React/Views/RCTView.m

```diff
@@ -341,6 +361,136 @@ - (NSString *)accessibilityValue
   }
   return nil;
 }
+#else // [TODO(macOS ISS#2323203)
+- (id)accessibilityValue {
+  if (self.accessibilityRole == NSAccessibilityCheckBoxRole) {
```

Comment:
radiobutton and disclosuretrianglerole should also have this YES NO boolean value right?

Reply:
yes, you're right.   And there's no way to set "disclosure" in react-native though.   We already have extended it for "list" and "group", so we could add "disclosure" for NSAccessibilityRoleDisclosureTriangle as well.